socorro/unittest/external/postgresql/test_crashstorage.py

```diff
@@ -266,7 +263,7 @@ def fetch_all_func(*args):
         crashstorage.save_processed(a_processed_crash)
 
         assert mocked_database_connection_source.call_count == 1
-        assert mocked_cursor.execute.call_count == 4
+        assert mocked_cursor.execute.call_count == 3
```

Comment:
We're not longer saving to the `processed_crashes_*` tables, so we can remove one call.